tianshou/highlevel/config.py

```diff
@@ -132,8 +150,8 @@ class SamplingConfig(ToStringMixin):
     to the agent for each time step. Setting this to a value greater than 1 can help agents learn
     temporal aspects (e.g. velocities of moving objects for which only positions are observed).
 
-    If the environment already stacks frames (e.g. using a `FrameStack` wrapper), this should either not
-    be used or should be used in conjunction with :attr:`replay_buffer_save_only_last_obs`.
```

Comment:
Why was the usage recommendation concerning the related parameter dropped?
Not clearly an improvement over the old docstring.

Reply:
I wanted to recommend to use FrameStack instead of using this option, previously this recommendation was not clear. I will restore the "in conjunction with :attr:`replay_buffer_save_only_last_obs`." part though

Reply:
Done, pls resolve if you agree with the new formulation